tests/model_registry/negative_tests/conftest.py

```diff
@@ -0,0 +1,121 @@
+import pytest
```

Comment:
as different features would have negative tests, wouldn't it be better to call this dire with a different name?

Reply:
For now there is no other negative tests for model registry and the test file already is unique enough.